### PR TITLE
ADD INTERACTION

### PR DESCRIPTION
**Pull request by emmanuellmhango**
The code was modified to add the following:

1. Interaction.js file which contains the following methods
       - Update status for tasks completed: true or false
       - Clear all completed tasks
2. Updated the index.js to import the interaction.js file as a module

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default class Todo {
       listItem.id = task.index.toString().concat('-task');
       const checkbox = document.createElement('input');
       checkbox.type = 'checkbox';
-      checkbox.checked = task.completed;
+      checkbox.id = task.index.toString().concat('-check');
       listItem.appendChild(checkbox);
```
Comment (Kossi-stack) [inline src/crud.js:50]: - [ ] Kindly create a module folder and import a javascript file containing methods related to the status updates (completed: true/false).

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @emmanuellmhango,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/crud.js (hunk 1 above)
Comment (emmanuellmhango) [reply src/crud.js:50]: Hi Kossi @Kossi-stack ,

Thank you for your input. However, I already have created a separate file, interaction.js in the src folder, and imported it into the index.js file. interaction.js is a class containing the methods of clearing the completed tasks and also the method to update the status of the task whether complete or not.

May you, therefore, clarify your instruction on exactly what I am expected to do?

Regards

**Review by aimalamiri — APPROVED** [review]
Hi @emmanuellmhango, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The app is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**